update upload page (#143)
dandi/handbook: docs/13_upload.md

   
     If the report is too large to efficiently navigate in your console, you can save a report using

        nwbinspector <source_folder> --config dandi --report-file-path <report_location>.txt
          
     For more details and other options, run:
                
        nwbinspector --help

   Thoroughly read the NWBInspector report and try to address as many issues as possible.
   **DANDI will prevent validation and upload of any issues labeled as level 'CRITICAL' or above when using the `--config dandi` option.**
    See 
       ["Validation Levels for NWB Files"](./135_validation.md) for more information about validation criteria for 
       uploading NWB 
       files and which are deemed critical. We recommend regularly running the inspector early in the process to generate the best NWB files possible. Note that some auto-detected violations, such as `check_data_orientation`, may be safely ignored in the event 
       that the data is confirmed to be in the correct form. See [the NWBInspector CLI documentation](https://nwbinspector.readthedocs.io/en/dev/user_guide/using_the_command_line_interface.html) for more information.

> 1. **Validate NWB files.** Once your files are confirmed to adhere to the Best Practices, perform an official validation of the NWB files by running: `dandi validate --ignore DANDI.NO_DANDISET_FOUND <source_folder>`.
>    **If you are having trouble with validation, make sure the conversions were run with the most recent version of `dandi`, `PyNWB` and `MatNWB`.**

Reviewer: Minor change to the format to display as a code block.

```suggestion
1. **Validate NWB files.** Once your files are confirmed to adhere to the Best Practices, perform an official validation of the NWB files by running:
        dandi validate --ignore DANDI.NO_DANDISET_FOUND <source_folder>
   **If you are having trouble with validation, make sure the conversions were run with the most recent version of `dandi`, `PyNWB` and `MatNWB`.**
```

1. **Upload the data to DANDI.** This can either be done through the NWB GUIDE, or programmatically:

        dandi download https://dandiarchive.org/dandiset/<dataset_id>/draft
        cd <dataset_id>
        dandi organize <source_folder> -f dry
        dandi organize <source_folder>
        dandi validate .
        dandi upload

   Note that the `organize` steps should not be used if you are preparing a BIDS dataset with the NWB files.
   Uploading to the development server is controlled via `-i` option, e.g.
   `dandi upload -i dandi-staging`.

>    Note that validation is also done during `upload`, but ensuring compliance using `validate` prior upload helps avoid interruptions of the lengthier upload process due to validation failures.

Reviewer: ```suggestion
   Note that validation is also done during `upload`, but ensuring compliance using `validate` prior to upload helps avoid interruptions of the lengthier upload process due to validation failures.
```

> 1. **Add metadata to Dandiset.** Visit your Dandiset landing page:

Reviewer: ```suggestion
1. **Add metadata to the Dandiset.** Visit your Dandiset landing page:
```

   `https://dandiarchive.org/dandiset/<dataset_id>/draft` and click on the `METADATA` link.

> If you have an issue using the Python CLI, see the [Dandi Debugging section](./15_debugging.md).

Reviewer: ```suggestion
If you have an issue using the DANDI Client, see the [DANDI Debugging section](./15_debugging.md).
```
